locales/en/messages.json

```diff
@@ -5225,6 +5225,10 @@
         "message": "Save to file",
         "description": "Save to file button in the VTX tab"
     },
+    "vtxButtonSaveLua": {
+        "message": "Save Lua Script",
```

Comment:
I think for this it would be helpful to add a tooltip to this button that provided a short explanation of how users can use this to get their custom VTX table supported in the lua scripts.

Reply:
Great idea... I'll also look at updating the lua readme.